View/home.php

```diff
@@ -0,0 +1,16 @@
+
+<?php
+
+  if (!isset($_SESSION))
```

Comment:
give braces for if like:
```suggestion
  if (!isset($_SESSION)) {
  }
```